src/foam/util/Emails/EmailsUtility.java

```diff
@@ -52,6 +56,8 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
         templateArgs = new HashMap<>();
         templateArgs.put("template", templateName);
       }
+      templateArgs.put("appConfig", appConfig);
+      templateArgs.put("theme", theme);
```

Comment:
again, both of these are incorrect.
the args must be key/value pairs. 
so if you need phonenumber then you would add
put("phoneNumber", theme.getSupportUser().getPhoneNumber())
for example.